packages/strapi-plugin-content-type-builder/controllers/ContentTypeBuilder.js

```diff
@@ -188,5 +188,24 @@ module.exports = {
     ctx.send({
       autoReload: _.get(strapi.config.environments, 'development.server.autoReload', false),
     });
+  },
+
+  checkTableExists: async ctx => {
+    // Get connec
+    const { connection } = ctx.params;
+    const model = _.toLower(ctx.params.model);
+    const connector = _.get(strapi.config.currentEnvironment.database.connections, [connection, 'connector']);
```

Comment:
What happens if the connector doesn't exist? You should return an error in this case.